.gitignore

```diff
@@ -2,3 +2,4 @@ target
 Cargo.lock
 !diesel_cli/Cargo.lock
 .env
+.DS_Store
```

Comment:
Please do not modify the project provided `.gitignore`. Such things belong into your system wide `.gitignore`.

Reply:
done